ui/analyse/src/view.ts

```diff
@@ -320,7 +320,7 @@ export function renderMaterialDiffs(ctrl: AnalyseCtrl): [VNode, VNode] {
 
   return materialView.renderMaterialDiffs(
     true, // showCaptured
-    ctrl.flipped,
+    ctrl.getOrientation() === 'black' ? true : false,
```

Comment:
This now doesn't work when the board is flipped locally (e.g. via the `F` key). You need to use `ctrl.flipped` as well.

Reply:
Are you sure? I tried this locally and it seems to work even when flipping the board. This `orientation` vs. `flipped` is very confusing IMO.

Reply:
Yes same for me, I just now tried with F and it works, as well as with the rotate button in the tool bar.

Reply:
Flipped was not set to true when the study chapters orientation is picked to be black, so this sets it as true in that case, and works the same as ctrl.flipped would in other cases.

Reply:
Yeah, you're right, my mistake, I checked the wrong thing.

Reply:
Although, please just change it to `ctrl.getOrientation() === 'black'`. There's no need for the `true` and `false` ...

Reply:
Done!